src/pages/Map.js

```diff
@@ -22,14 +24,28 @@ export default function Map() {
     });
   };
 
+  //TODO: added just in case we want to access the googleMapApiInternals
+  const handleApiLoaded = (map, maps) => {
+    // use map and maps objects
+  };
+
+  /*When the map moves it resets the center to recenter 
+  we are updating the state with the setRecenter which updates context and updates locationData */
+  const handleChange = (e) => {
+    setRecenter({ lat: e.center.lat, lng: e.center.lng });
+  };
+
   return (
     <div className="map">
       <GoogleMapReact
         bootstrapURLKeys={{ key: API_KEY }}
+        center={recenter}
```

Comment:
Ah, OK, I see how this is being used now. What would you think of renaming `recenter` to `newCenter`?